pkgs/development/libraries/at-spi2-core/default.nix

```diff
@@ -62,6 +62,11 @@ stdenv.mkDerivation rec {
     glib
   ];
 
+  patches = [
+    # Fix implicit declaration of `strcasecmp`, which is an error on clang 16.
+    ./fix-implicit-strcasecmp.patch
```

Comment:
Could you please forward this [upstream](https://gitlab.gnome.org/GNOME/at-spi2-core) and then use `fetchpatch` here?